### PR TITLE
Feature/multiple services one yaml

### PR DESCRIPTION
**Pull request by Anonycoders**

### CODE REVIEW

File: fandogh_cli/service_commands.py [hunk 1]
```diff
@@ -171,53 +173,63 @@ def service_apply(file, parameters, detach):
...
-    Since your service is internal, it's not accessible from outside your fandogh private network, 
-    but other services inside your private network will be able to find it using it's name: '{}'
-            """.strip().format(
-            deployment_result['name']
-        )
-    elif service_type == 'managed':
-        message += """
-        Managed service deployed successfully
-        """
+    for index, service_conf in enumerate(manifest_content):
```
Comment (MahdiZareie) [inline fandogh_cli/service_commands.py]: wouldn't be nice if we merge this for with the previous one and display a message like this?
```
Deploying following manifest:
kind: ExternalService
name: hello-felan
spec:
  blah
  blah
  blah

```

File: fandogh_cli/utils.py [hunk 2]
```diff
@@ -85,17 +85,17 @@ def trim_comments(manifest):
 
 def read_manifest(manifest_file, parameters):
     try:
-        deployment_list = list()
+        service_config_list = list()
         with open(manifest_file, mode='r') as manifest:
-            for deployment in manifest.read().split('---'):
+            for config in manifest.read().split('---'):
```
Comment (MahdiZareie) [inline fandogh_cli/utils.py]: Why not just standard yaml array?
```
- kind: ExternalService
  name: foo
  spec:
    image: api:20
- kind: ManagedService
  name: db
  spec:
    service_name: mysql:5.6

- kind: InternalService
  name: cache
  spec:
    image: redis:latest

```

File: fandogh_cli/service_commands.py (hunk 1 above)
Comment (Anonycoders) [reply fandogh_cli/service_commands.py]: Good idea

File: fandogh_cli/utils.py (hunk 2 above)
Comment (Anonycoders) [reply fandogh_cli/utils.py]: It's instructed from the product designer

Comment (MahdiZareie) [reply fandogh_cli/utils.py]: What is the reason behind this decision?
@psycho-ir

**Review by s-soroosh — APPROVED** [review]

**Pull request merged**